### PR TITLE
fix(sentry-ruby): A note on Rack apps without Rackup

### LINKED ISSUE #8519: Clarify sinatra usage
Would that be worth putting in docs, since that came up a couple of times? Perhaps here? https://docs.sentry.io/platforms/ruby/guides/rack/

_Originally posted by @natikgadzhi in https://github.com/getsentry/sentry-ruby/issues/2167#issuecomment-1815705269_

### PR DESCRIPTION
**Pull request by natikgadzhi**
## Summary

This pull request adds a note on using Sentry in Rack apps that are run directly with Ruby, without Rackup. Closes #8519. Related to [this Sentry-Ruby issue](https://github.com/getsentry/sentry-ruby/issues/2167).

/cc @sl0thentr0py

Screenshot 2023-11-22 at 11 38 00 AM

## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

*Describe your changes here. If your PR relates to or resolves an issue, add a link to that too.*

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by sl0thentr0py — APPROVED** [review]
ty @natikgadzhi !

**Comment by natikgadzhi** [comment]
That was quick ;-)

File: src/platform-includes/getting-started-config/ruby.rack.mdx [hunk 1]
```diff
@@ -20,3 +22,29 @@ end
...
+
+```Ruby {filename:app.rb}
+require 'sentry-ruby'
+
+Sentry.init do |config|
+  config.dsn = '___PUBLIC_DSN___'
+  config.breadcrumbs_logger = [:sentry_logger, :http_logger]
+end
+
+# Notice: requiring sinatra must go after Sentry SDK is initialized in a non-rackup environment
```
Comment (shanamatthews) [inline src/platform-includes/getting-started-config/ruby.rack.mdx:43]:
```suggestion
# in a non-rackup environment you must initialize the Sentry SDK before requiring sinatra
```

This phrasing just sounded weird to me, so optional rephrase suggestion

**Review by shanamatthews — APPROVED** [review]
Good from me!

**Comment by shanamatthews** [comment]
@natikgadzhi - I'll merge this once you lmk it's good to go. Thanks for the contribution!

**Comment by natikgadzhi** [comment]
LGTM!

**Pull request merged**